crates/medmodels-core/src/medrecord/group_mapping.rs

```diff
@@ -8,21 +8,26 @@ pub type Group = MedRecordAttribute;
 #[derive(Debug, Serialize, Deserialize)]
 pub(super) struct GroupMapping {
     nodes_in_group: MrHashMap<Group, MrHashSet<NodeIndex>>,
+    edges_in_group: MrHashMap<Group, MrHashSet<EdgeIndex>>,
     groups_of_node: MrHashMap<NodeIndex, MrHashSet<Group>>,
+    groups_of_edge: MrHashMap<EdgeIndex, MrHashSet<Group>>,
 }
 
 impl GroupMapping {
     pub fn new() -> Self {
         Self {
             nodes_in_group: MrHashMap::new(),
+            edges_in_group: MrHashMap::new(),
             groups_of_node: MrHashMap::new(),
+            groups_of_edge: MrHashMap::new(),
         }
     }
 
     pub fn add_group(
         &mut self,
         group: Group,
         node_indices: Option<Vec<NodeIndex>>,
+        edge_indices: Option<Vec<EdgeIndex>>,
```

Comment:
Does this assume node_indices or edge_indices are None if not specified? Or do they always need to be indicated?

Reply:
Nope. There are no default function argument values in rust so every argument has to be specified, in this case by using `None` or `Some(...)`